packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -133,16 +128,6 @@ export function mapTreeFromNodeData(
 	// Add what defaults can be provided. If no `context` is providing, some defaults may still be missing.
 	addDefaultsToMapTree(mapTree, normalizedFieldSchema.allowedTypes, context);
 
-	if (schemaValidationPolicy?.policy.validateSchema === true) {
-		// TODO: BUG: AB#9131
```

Comment:
This bug is fixed by this change. Validation has moved to where it can always work.